src/components/Footer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import '../css/Footer.scss';
+
+class Footer extends React.Component {
+  render() {
+    return (
```

Comment:
This should be a stateless, functional component.

```suggestion
const Footer = () => (
  ...
)
```